### PR TITLE
 Add Alexa Bobina to the student list .md

### PR DESCRIPTION
**Pull request by alexabobina**
Add Alexa Bobina to the student list

### CODE REVIEW

**Comment by ava8katushka** [comment]
Please, rename the pull request from Update README.md  to Add Your Name to the student list and resolve the conflicts.

**Comment by alexabobina** [comment]
Updated :)

**Comment by vtknann** [comment]
Why did you close it?

**Comment by alexabobina** [comment]
oops( I thought that's how I save my changes. Sorry for that

File: README.md [hunk 1]
```diff
@@ -12,5 +12,6 @@
 | 2 | Веня Стаховский | Интерстеллар                           |
 | 3 | Михаил Антаманов| Питер-ФМ                               |
 | 4 | Ири Айларова    | Звездные войны                         |
+| 5 | Алекса Бобина   | Вечное сияние чистого разума           |
```
Comment (ava8katushka) [inline README.md]: Сейчас получается два номера 5. Можешь добавить себя в конец списка?

**Review by ava8katushka — CHANGES_REQUESTED** [review]

**Comment by ava8katushka** [comment]
I merged your pull request, but noticed that you renamed the file README.md. 
The file should stay named README.md

**Comment by alexabobina** [comment]
My bad(

The name is correct now, I assume you've changed it yourself, thank you.
Please let me know if there is any action item remaining on me.




ср, 24 февр. 2021 г. в 17:35, ava8katushka <notifications@github.com>:

**Comment by ava8katushka** [comment]
It's all good now, I fixed, no action item on you :)

**Pull request merged**